### PR TITLE
Bug-batching: CVO 4.16 bug fix release notes

### PR DESCRIPTION
**Pull request by obrown1205**
Version(s):
4.16

Issue:
N/A

Link to docs preview:
https://77670--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-bug-fixes_release-notes

QE review:
N/A

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -1060,6 +1060,12 @@ With this release, the CCO no longer checks for the nonexistent permission.
 [id="ocp-4-16-cluster-version-operator-bug-fixes_{context}"]
 ==== Cluster Version Operator
 
+* This release expands the `ClusterOperatorDown` and `ClusterOperatorDegraded` alerts to cover ClusterVersion conditions and send alerts for `Available=False` (`ClusterOperatorDown`) and `Failing=True` (`ClusterOperatorDegraded`). In previous releases, those alerts only covered ClusterOperator conditions. (link:https://issues.redhat.com/browse/OCPBUGS-9133[*OCPBUGS-9133*])
+
+* Previously, Cluster Version Operator (CVO) changes that were introduced in {product-title} 4.15.0, 4.14.0, 4.13.17, and 4.12.43 caused failing risk evaluations to block the CVO from fetching new update recommendations. When the risk evaluations failed, the bug caused the CVO to overlook the update recommendation service.  With this release, the CVO continues to poll the update recommendation service, regardless of whether update risks are being successfully evaluated and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-25708[*OCPBUGS-25708*])
```
Comment (tmalove) [inline release_notes/ocp-4-16-release-notes.adoc:1065]:
```suggestion
* Previously, Cluster Version Operator (CVO) changes that were introduced in {product-title} 4.15.0, 4.14.0, 4.13.17, and 4.12.43 caused failing risk evaluations to block the CVO from fetching new update recommendations. When the risk evaluations failed, the bug caused the CVO to overlook the update recommendation service. With this release, the CVO continues to poll the update recommendation service, regardless of whether update risks are being successfully evaluated and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-25708[*OCPBUGS-25708*]) 
```
Removed extra space.

**Review by tmalove — APPROVED** [review]

File: release_notes/ocp-4-16-release-notes.adoc (hunk 1 above)
Comment (obrown1205) [reply release_notes/ocp-4-16-release-notes.adoc]: good catch! thank you! :)

File: release_notes/ocp-4-16-release-notes.adoc [hunk 2]
```diff
@@ -1060,6 +1060,12 @@ With this release, the CCO no longer checks for the nonexistent permission.
...
+
+* Previously, Cluster Version Operator (CVO) changes that were introduced in {product-title} 4.15.0, 4.14.0, 4.13.17, and 4.12.43 caused failing risk evaluations to block the CVO from fetching new update recommendations. When the risk evaluations failed, the bug caused the CVO to overlook the update recommendation service. With this release, the CVO continues to poll the update recommendation service, regardless of whether update risks are being successfully evaluated and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-25708[*OCPBUGS-25708*]) 
+
+* Previously, the CVO was unable to find and download release signatures for standalone clusters that were running 4.16.0-rc.3 and 4.16.0-rc.4 with a default feature set. With this release, the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-35249[*OCPBUGS-35249*])
```
Comment (skopacz1) [inline release_notes/ocp-4-16-release-notes.adoc:1067]: Should this be included in the RNs? It's talking about issues on release candidate builds, and is essentially a bug that affected only 4.16+ before 4.16 was GA, meaning no customers were affected by the bug. 

I think in the RN FAQ doc it says that bugs with only 4.16 as both the affect and fix versions can be marked as Release Note Not Required for those reasons.
```suggestion
```

**Review by skopacz1 — COMMENTED** [review]
Just one comment, let me know if it's addressed or not-applicable and I can go ahead and merge

File: release_notes/ocp-4-16-release-notes.adoc (hunk 2 above)
Comment (obrown1205) [reply release_notes/ocp-4-16-release-notes.adoc:1067]: good catch! I think you're right. I'll remove this. :)

**Pull request merged**